android/src/main/java/com/iterable/reactnative/RNIterableAPIModule.java

```diff
@@ -210,6 +210,19 @@ public void execute(@Nullable Uri url) {
         });
     }
 
+    @ReactMethod
+    public void getHtmlInAppContentForMessage(String messageId, final Promise promise) {
+        IterableLogger.printInfo();
+        IterableInAppMessage message = RNIterableInternal.getMessageById(messageId);
+        if (message == null) {
```

Comment:
Should reject promise here.

Reply:
We should always either resolve or reject promise.

Reply:
Yes.. I will look up in iOS counterpart what to send in reject in case of failure..